### ISSUE TITLE
gurobipy_pandas methods are not exported for pyright type checker

### ISSUE THREAD

**Issue by Dr-Irv** [Gurobi/gurobipy-pandas#73]
See the pic of using pylance/pyright in VS Code:
![image](https://github.com/Gurobi/gurobipy-pandas/assets/15113894/39c37b16-ae78-4206-b8db-cd49ae157d7c)

In `gurobipy_pandas/__init__.py`, if I change to this line:
```python
from gurobipy_pandas.api import add_vars as add_vars # noqa: F401
```
Then the error goes away.

**Comment by simonbowly** [comment]
I can't reproduce the issue in my vscode or running pyright manually (pyright 1.1.363) but would prefer to fix it by defining:

```
__all__ = ["add_vars", "add_constrs", "set_interactive"]
```

as the redundancy seems really odd to me. According to [pyright's docs](https://github.com/microsoft/pyright/blob/main/docs/typed-libraries.md#library-interface) this should also do the trick.

**Comment by simonbowly** [comment]
This fix is released with version 1.1.2
